specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/Builds.json

```diff
@@ -0,0 +1,559 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./Builders.json#/parameters/BuilderNameParameter"
+          },
+          {
+            "$ref": "#/parameters/BuildNameParameter"
+          },
+          {
+            "name": "buildEnvelope",
+            "in": "body",
+            "required": true,
+            "description": "Resource create or update parameters.",
+            "schema": {
+              "$ref": "#/definitions/BuildResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ARM create or update operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/BuildResource"
+            }
+          },
+          "201": {
+            "description": "ARM create or update operation has been started.",
+            "schema": {
+              "$ref": "#/definitions/BuildResource"
+            },
+            "headers": {
+              "Azure-AsyncOperation": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Builds_CreateOrUpdate_WithConfig": {
+            "$ref": "./examples/Builds_CreateOrUpdate.json"
+          },
+          "Builds_CreateOrUpdate_NoConfig": {
+            "$ref": "./examples/Builds_CreateOrUpdate_NoConfig.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Builds"
+        ],
+        "operationId": "Builds_Delete",
+        "description": "Delete a BuildResource",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./Builders.json#/parameters/BuilderNameParameter"
+          },
+          {
+            "$ref": "#/parameters/BuildNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
The long running delete operation should not contain 200 status code.

refs: https://github.com/Azure/azure-openapi-validator/blob/main/docs/delete-response-codes.md